examples/exploring_priors.jl

```diff
@@ -0,0 +1,105 @@
+# # Exploring priors
+#
+# This example explores specifying priors using OceanTurbulenceParameterEstimation. 
+#
+# ## Install dependencies
+#
+# First let's make sure we have all required packages installed.
+
+# ```julia
+# using Pkg
+# pkg"add OceanTurbulenceParameterEstimation, Oceananigans, Distributions, CairoMakie"
+# ```
+
+# First we load few things
+
+using OceanTurbulenceParameterEstimation
+using CairoMakie
+using Distributions
+
+# `OceanTurbulenceParameterEstimation` supports three types of prior
+# distributions. The normal prior distribution,
+
+normal_prior = Normal(0.1, 0.1)
+
+# the log-normal prior,
+
+lognormal_prior = lognormal(mean=0.1, std=0.1)
+
+# and the "scaled", logit-normal prior,
+
+logitnormal_prior = ScaledLogitNormal(bounds=(0, 0.2))
+
+# Note that `lognormal` is a constructor for `Lognormal(μ, σ)` that calculates
+# the parameters `μ` and `σ` given a target distribution `mean` and 
+# standard deviation `std`.
+#
+# Sampling the distributions reveals their properties:
+
+samples = 10^6
+fig = Figure()
+ax = Axis(fig[1, 1:9], xlabel="Random samples of priors", ylabel="Density")
+
+density!(ax, rand(normal_prior, samples), label="Normal")
+density!(ax, rand(lognormal_prior, samples), label="Log-normal")
+density!(ax, rand(logitnormal_prior, samples), label="Scaled logit-normal")
+
+axislegend(ax)
+xlims!(ax, -0.5, 1.0)
+
+save("prior_flavors.svg", fig); nothing #hide 
+# ![](prior_flavors.svg)
+
+# We note three important features:
+#
+# 1. `Normal` samples can be negative.
+# 2. `Lognormal` samples cannot be negative, but can have large positive values.
+# 3. `ScaledLogitNormal` samples are _bounded_.
+#
+# Boundedness is a very useful property of `ScaledLogitNormal`, so we
+# explore specifying `ScaledLogitNormal` priors in more detail.
+#
+# @doc ScaledLogitNormal
```

Comment:
What is this supposed to be? Look how it is rendered at
https://clima.github.io/OceanTurbulenceParameterEstimation.jl/previews/PR174/literated/exploring_priors/

Reply:
Got rid of it